custom_components/tuya_local/devices/its_45hd_heatpump.yaml

```diff
@@ -58,20 +58,48 @@ primary_entity:
     - id: 20
       type: integer
       name: compressor_strength
-    - id: 26
+    - id: 21
       type: integer
       name: current_temperature
     - id: 28
       type: boolean
       name: four_valve_state
       hidden: true
+    - id: 30
+      type: boolean
+      name: Chlorination
+      attribute: true
     - id: 32
       type: boolean
       name: aux_heat
       readonly: true
     - id: 40
       type: integer
       name: model_code
+    - id: 101
+      type: integer
+      name: Current
+      attribute: true
+    - id: 102
+      type: integer
+      name: Voltage
+      attribute: true
+    - id: 103
+      type: integer
+      name: Power
+      attribute: true
+    - id: 104
+      type: integer
+      name: Energy
+      attribute: true
+    - id: 105
+      type: boolean
+      name: Electricity monitor
```

Comment:
One further note: at attribute (dps) level, the name is not a UI exposed name (Home Assistant automatically capitalises and removes underscores - the opposite of what it does to entity names) so it should have underscores in place of spaces and by convention be all lower case.

Reply:
The lower case part of that also applies to the other attributes converted from sensors by the last commit.

Reply:
Thank you for your patience and educating me. Its still new to me. Will submit update shortly

Reply:
I noticed that I previously removed "Electricty today" sensor, but forgot to add the attribute to the climate entity. Will add it now and send upadte